Gordon360/Models/ViewModels/AdvisorViewModel.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using System.Web;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class AdvisorViewModel
+    {
+        public AdvisorViewModel(string fname, string lname, string adname)
+        {
+            Firstname = fname;
+            Lastname = lname;
+            ADUserName = adname;
+        }
+        public string Firstname { get; set; }
+        public string Lastname { get; set; }
+        public string ADUserName { get; set; }
+    }
+
+    // Collection of Advisor objects. This class
```

Comment:
Since your service uses a List<AdvisorViewModel>, I don't think the rest of this file is needed.  I think you can delete it, and the line I asked about in your service.  If you think I'm wrong, please help me understand why it's needed.